docs/reference/search/aggregations/reducers/movavg-reducer.asciidoc

```diff
@@ -0,0 +1,296 @@
+{
+    "movavg": {
+        "buckets_path": "the_sum",
+        "model": "double_exp",
+        "window": 5,
+        "gap_policy": "insert_zero",
+        "settings": {
+            "alpha": 0.8
+        }
+    }
+}
+--------------------------------------------------
+
+.`moving_avg` Parameters
+|===
+|Parameter Name |Description |Required |Default
+
+|`buckets_path` |The path to the metric that we wish to calculate a moving average for |Required |
+|`model` |The moving average weighting model that we wish to use |Optional |`simple`
+|`gap_policy` |Determines what should happen when a gap in the data is encountered. |Optional |`insert_zero`
+|`window` |The size of window to "slide" across the histogram. |Optional |`5`
+|`settings` |Model-specific settings, contents which differ depending on the model specified. |Optional |
+|===
+
+
+`moving_avg` aggregations must be embedded inside of a `histogram` or `date_histogram` aggregation.  They can be
+embedded like any other metric aggregation:
+
+[source,js]
+--------------------------------------------------
+{
+    "my_date_histo":{                <1>
+        "date_histogram":{
+            "field":"timestamp",
+            "interval":"day",
+            "min_doc_count": 0      <2>
+        },
+        "aggs":{
+            "the_sum":{
+                "sum":{ "field": "lemmings" } <3>
+            },
+            "the_movavg":{
+                "moving_avg":{ "buckets_path": "the_sum" } <4>
+            }
+        }
+    }
+}
+--------------------------------------------------
+<1> A `date_histogram` named "my_date_histo" is constructed on the "timestamp" field, with one-day intervals
+<2> We must specify "min_doc_count: 0" in our date histogram that all buckets are returned, even if they are empty.
+<3> A `sum` metric is used to calculate the sum of a field.  This could be any metric (sum, min, max, etc)
+<4> Finally, we specify a `moving_avg` aggregation which uses "the_sum" metric as it's input.
+
+Moving averages are built by first specifying a `histogram` or `date_histogram` over a field.  You can then optionally
+add normal metrics, such as a `sum`, inside of that histogram.  Finally, the `moving_avg` is embedded inside the histogram.
+The `buckets_path` parameter is then used to "point" at one of the sibling metrics inside of the histogram.
+
+A moving average can also be calculated on the document count of each bucket, instead of a metric:
+
+[source,js]
+--------------------------------------------------
+{
+    "my_date_histo":{
+        "date_histogram":{
+            "field":"timestamp",
+            "interval":"day",
+            "min_doc_count": 0
+        },
+        "aggs":{
+            "the_movavg":{
+                "moving_avg":{ "buckets_path": "_count" } <1>
+            }
+        }
+    }
+}
+--------------------------------------------------
+<1> By using `_count` instead of a metric name, we can calculate the moving average of document counts in the histogram
```

Comment:
Does it work on all reducers? If yes I think it should have its own section in the documentation of the path syntax instead of being here?